image/Dockerfile

```diff
@@ -26,23 +31,38 @@ RUN make build-linux
 
 # --------------------------------------------------------------------------------------------------
 # The downloading stage.
-FROM alpine:latest AS downloader
+FROM ${DOWNLOADER_IMAGE} AS downloader
 
-ENV HELM_VERSION="v2.13.1"
-ENV FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
+ARG HELM_VERSION="v2.16.7"
+ARG HELM_FILENAME="helm-${HELM_VERSION}-linux-amd64.tar.gz"
```

Comment:
Does this really need to be configurable? If yes, is there a point in making the version (which is only used here) configurable?

Reply:
Yes, I want to eventually have a single source of truth for the Helm version. Right now, it's hardcoded in more than one place. I don't think it hurts making it configurable now.

Reply:
I made the `HELM_FILENAME` an env var rather than a build arg. https://github.com/kubernetes-sigs/minibroker/pull/53/commits/ca7c833d628f43051a546eaede6a494d86dcdc0c